requirements.in

```diff
@@ -16,3 +16,6 @@ pyjwt
 requests-futures
 # we require f8a_worker, but it is installed manually as it is not available in PyPI
 #f8a_worker
+
+google-cloud
+joblib
```

Comment:
Given time constrains, I am going to open an issue for this. I think we will be able to just drop those requirements in another PR.